<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>614</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Okcham Data Mining System</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tasks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_ETL">
     <attribute name="title">
      <string>ETL</string>
     </attribute>
     <widget class="QPushButton" name="button_run_ETL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ETL</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_etl_process">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>391</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_schedule">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run schedule:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_minute">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Minute:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_hour">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Hour:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_month_day">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>150</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Day of month:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_week_day">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Day of week:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_minute">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_hour">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>23</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="button_set_etl_cron">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>210</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_month_day">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>150</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_month">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Month:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_month">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="button_cron_notation">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>290</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Cron Notation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cron">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>260</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cron:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_cron">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>260</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_week_day">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>180</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_OLAP">
     <attribute name="title">
      <string>OLAP</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>641</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_x">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_x">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_y">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>y:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_y">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_z_value">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_z">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_olap">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Data Mining</string>
     </attribute>
     <widget class="QPushButton" name="button_classify">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Classify</string>
      </property>
     </widget>
     <widget class="QListView" name="list_clasterized">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>631</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_clasterize">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clasterize</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_frequency">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequency:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_frequency">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_classification">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>631</width>
        <height>201</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>Low</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Middle</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>High</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>391</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Perfomrs suppliers classification by risk</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
